pkg/server/controller.go

```diff
@@ -256,6 +256,15 @@ func (ctrl *Controller) serverMux() (http.Handler, error) {
 		ctrl.drainMiddleware,
 		ctrl.authMiddleware(nil))
 
+	deleteAppRouter := r.Path("/delete-app").Subrouter()
```

Comment:
For consistency sake I would suggest to use ´DELETE /apps/`
https://github.com/pyroscope-io/pyroscope/blob/main/pkg/admin/server.go#L42